hikari/audit_logs.py

```diff
@@ -406,6 +406,7 @@ async def fetch_channel(self) -> channels.GuildVoiceChannel:
         return channel
 
 
+@attr.s(eq=False, hash=False, init=False, slots=False, str=False, repr=False)
```

Comment:
What's the point of this?

Reply:
Was testing something out, forgot to remove it